### ISSUE TITLE
🐛 BUG: Kindness cause cards can't be selected by keyboard

### ISSUE THREAD

**Issue by RafidMuhymin** [dilmaheu/dilmahtea.me#242]
**Describe the bug**
Kindness cause cards can't be selected through tab navigation.

**Steps to Reproduce**

1. Go to https://dilmahtea.me/en/checkout/kindness/
2. Go to a card through tab navigation 
3. See that nothing happens after pressing the `Enter` key.

**Expected behavior**
It should be possible to select a card by keyboard.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 110

**Additional context**
N/A
